DataManager/src/Tests/VersionTester.java

```diff
@@ -0,0 +1,73 @@
+package Tests;
+
+import java.awt.BorderLayout;
+import java.awt.Component;
+import java.awt.Rectangle;
+
+import javax.swing.JFrame;
+import javax.swing.JTextPane;
+
```

Comment:
Would using a mocking framework to "mock" the environment and using unit tests to whitebox test the code be better or this?